From d2347e0ee69510115cf164af471c9daabdb9662a Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Fri, 20 Oct 2017 23:55:34 +0800
Subject: [PATCH 2672/5242] MLK-16676-4 ARM64: dts: freescale: imx8qm: add
 debug_console property

commit  c6ce78e6ea0adf9cce3421fe4ed84cd737a692b5 from
https://source.codeaurora.org/external/imx/linux-imx.git

Pass debug_console port info for power domain driver awareness.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts  |    4 ++++
 arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts   |    4 ++++
 2 files changed, 8 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
index 115bea2..760779b 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
@@ -739,6 +739,10 @@
         };
 };
 
+&pd_dma_lpuart0 {
+	debug_console;
+};
+
 &lpuart0 { /* console */
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_lpuart0>;
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts
index 4bd5913..3d0909b 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts
@@ -445,6 +445,10 @@
 	status = "okay";
 };
 
+&pd_dma_lpuart0 {
+	debug_console;
+};
+
 &lpuart0 { /* console */
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_lpuart0>;
-- 
1.7.9.5

